### DISCUSSION TITLE
Change Behavior of ADD TO CART feature on Product Image

### DISCUSSION THREAD

**Discussion by AlohaJeff** [breezefront/community#8] [1. Help]
What would the preferred method be to remove the ADD TO CART button that appears in the center of a product image when listed in the Catalog? On a mobile device users are clicking the image to view the product and that in many cases adds the product to the cart but it does not always direct them to the product page. Sometimes the user is tapping the image multiple times which adds more of the item to the cart and it frustrates the user as that's not the behavior they were expecting. For our use the ADD TO CART feature overlay on the image would be removed and clicking on the product picture would only be responsible for actually viewing the product.

Thank you!

**Comment by vovayatsyuk** [comment]
1. Hmm.. This should not be the case with a touch device. We don't show the "add to cart" button in the listing when a user uses a phone or tablet exactly because of this reason. What device do you use?
2. It's done with less variables: https://github.com/breezefront/theme-frontend-breeze-evolution/blob/master/web/css/theme/abstracts/_variables.less#L171-L173. Change them to 'static' to get default "Breeze Blank" behavior.

**Reply by AlohaJeff** [reply]
We have tested with a few different devices.
Android = Samsung Internet / Chrome
Apple iPhone = Safari

It also does it on mouseover with Chrome, FireFox and Edge. After further testing we would like to remove the ADD TO CART functionality on hover / mouseover on all devices regardless of resolution and/or browser type.
![add-to-cart](https://user-images.githubusercontent.com/113044284/188995512-c77e58cd-fb08-48d4-92b0-bd4204bcce35.jpg)

**Reply by vovayatsyuk** [reply]
As I already said, this was made with less variables. Please revert them to default values.

**Comment by AlohaJeff** [comment]
Thank you! I made that change and ran rm -rf var/cache/* var/view_preprocessed/* generated/* pub/static/* and then deployed the content again and the ADD TO CART button is now hidden. That less file was never edited previously by anyone so I'm not sure how the default action would be changed.
